src/olympia/addons/migrations/0044_remove_addonreviewerflags_notified_about_auto_approval_delay_and_more.py

```diff
@@ -0,0 +1,184 @@
+# Generated by Django 4.2.1 on 2023-05-24 08:34
+
+from django.db import migrations, models
+import django.db.models.deletion
+import olympia.translations.fields
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('translations', '0001_initial'),
+        ('addons', '0043_addonreviewerflags_auto_approval_delayed_until_unlisted'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='addonreviewerflags',
+            name='notified_about_auto_approval_delay',
+        ),
```

Comment:
Note: this is the only one that will do anything in the database (which is fine)